### PR TITLE
Update report.py

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/api/report.py [hunk 1]
```diff
@@ -210,6 +212,47 @@ def week_checkin_reports():
...
+def revoke_checkin_reports():
+    current_user = get_current_user()
+    date_t=current_user["revoke"]
+    today = date_t
+    last_sunday = today - datetime.timedelta(days=(today.weekday() + 1))
+    last_monday = today - datetime.timedelta(days=(today.weekday() + 8))
+    current_user = get_current_user()
+    docs = mongo.db.reports.find({
+        "user": str(current_user["_id"]),
+        "type": "daily",
```
Comment (reactexcel) [inline app/api/report.py]: why are we checking daily report.  this has nothing to do with daily reports

File: app/api/report.py [hunk 2]
```diff
@@ -210,6 +212,47 @@ def week_checkin_reports():
...
+            "revoke":datee
+        }
+    },upsert=True)
+
+    docs = mongo.db.reports.remove({
+        "_id": ObjectId(weekly_id),
+        "type": "weekly",
+    })
+    return jsonify(str(docs))
+
```
Comment (reactexcel) [inline app/api/report.py]: where is other logic of allowing user to update report

Comment (pythexcel) [reply app/api/report.py]: user can submit weekly with same API

File: app/api/report.py (hunk 1 above)
Comment (pythexcel) [reply app/api/report.py]: this is because we are showing last to last checkins on weekly submit page.

**Pull request merged**